store/sqlstore/command_store.go

```diff
@@ -91,54 +96,56 @@ func (s SqlCommandStore) GetByTrigger(teamId string, trigger string) (*model.Com
 		query = "SELECT * FROM Commands WHERE TeamId = :TeamId AND \"trigger\" = :Trigger AND DeleteAt = 0"
 	}
 
-	if err := s.GetReplica().SelectOne(&command, query, map[string]interface{}{"TeamId": teamId, "Trigger": trigger}); err != nil {
-		return nil, model.NewAppError("SqlCommandStore.GetByTrigger", "store.sql_command.get_by_trigger.app_error", nil, "teamId="+teamId+", trigger="+trigger+", err="+err.Error(), http.StatusInternalServerError)
+	if err := s.GetReplica().SelectOne(&command, query, map[string]interface{}{"TeamId": teamId, "Trigger": trigger}); err == sql.ErrNoRows {
+		return nil, store.NewErrNotFound("Command", trigger)
```

Comment:
Let's also add the `teamId` while we are here. Perhaps make a string concatenation to create the final string like done in the original code.

Reply:
Also I am still not sure on this one even though I've put up a patch. Looking forward to hearing on this more :slightly_smiling_face: